src/main/java/com/mlaf/hu/decisionagent/behavior/RegisterBehavior.java

```diff
@@ -0,0 +1,31 @@
+package com.mlaf.hu.decisionagent.behavior;
+
+import com.mlaf.hu.decisionagent.DecisionAgent;
+import com.mlaf.hu.models.InstructionSet;
+import jade.core.behaviours.CyclicBehaviour;
+import jade.lang.acl.ACLMessage;
+import jade.lang.acl.MessageTemplate;
+
+public class RegisterBehavior extends CyclicBehaviour{
```

Comment:
Misschien handig (Niet alleen voor deze code, maar ook voor mn eigen) om bovenaan elke behavior te zetten in korte documentatie wat het doel is van elke behavior?

Reply:
Zeker voor een framework zijn dat soort dingen belangrijk. Ga ik doen.